Wiki/BlankConfig/index.md

```diff
@@ -0,0 +1,44 @@
+---
+layout : wiki
+title : BlankConfig
+type : config
+weight : 500
+description : |
+  This is a description of how to configure Fvwm3 from scratch with a blank config file
+---
+
+* TOC
+{:toc}
+
+# Starting Fvwm3 with a Blank Config File
+
+Fvwm has been written with literally hundreds of options for most (if not all) aspects of window management.
+
+In order for windows to appear in the first place, there needs to be a root area (which, in turn, needs to "look like something"); a window needs to either have decorations or no decorations; if it has decorations, they need to be defined and styled; once a window is opened, it needs to open with some kind of geometry somewhere over the root area, etc...
```

Comment:
When you say "there needs to be a root area", what are you referring to here?  The root window?

Reply:
Yes, I mean "root window". I probably was oversimplifying and tried to avoid "window" in this context. But people who are looking into Fvwm3 in the first place are not entirely clueless, so my didactic reduction to "root area" will be replaced by the correct term "root window". :)

Reply:
Done.